content/en/real_user_monitoring/application_monitoring/ios/advanced_configuration.md

```diff
@@ -748,6 +753,49 @@ URLSessionInstrumentation.disable(delegateClass: <YourSessionDelegate>.self)
 {{% /tab %}}
 {{< /tabs >}}
 
+#### Apollo instrumentation
+
+1. [Set up][2] RUM monitoring with Datadog iOS RUM.
+
+2. [Set up](#basic-network-instrumentation) URLSession instrumentation with the Datadog RUM SDK.
+
+3. Add the following to your `Package.swift` file:
+
+```swift
+dependencies: [
+    .package(url: "https://github.com/DataDog/dd-sdk-ios-apollo-interceptor", .upToNextMajor(from: "1.0.0"))
+]
+```
```

Comment:
If you want the code block to align with the numbering:

````suggestion
3. Add the following to your `Package.swift` file:

   ```swift
   dependencies: [
       .package(url: "https://github.com/DataDog/dd-sdk-ios-apollo-interceptor", .upToNextMajor(from: "1.0.0"))
   ]
   ```
````